npm/react/plugins/babel/index.js

```diff
@@ -18,3 +18,7 @@ module.exports = getLegacyDevServer(devServer, (config) => {
 
 // New signature
 module.exports.devServer = devServer
+
+module.exports.defineDevServerConfig = function (config) {
```

Comment:
For consistency, the first argument to defineDevServerConfig should be called devServerConfig in all modules.